libraries/lib-graphics/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#[[
+A library responsible for custom rendering in Audacity
+]]#
+
+set( SOURCES
+   FrameStatistics.cpp
+   FrameStatistics.h
+
+   GraphicsDataCache.cpp
+   GraphicsDataCache.h
+
+   PixelSampleMapper.cpp
+   PixelSampleMapper.h
+
+   waveform/WaveData.cpp
+   waveform/WaveData.h
+   waveform/WaveDataCache.cpp
+   waveform/WaveDataCache.h
+)
+set( LIBRARIES
+   PUBLIC
+      lib-utility-interface
+   PRIVATE
+      lib-math-interface
+      lib-screen-geometry-interface
+      wxwidgets::base
```

Comment:
At the later commit that adds more dependencies, I would simplify:

```
set( LIBRARIES
       lib-math-interface
       lib-screen-geometry-interface
PRIVATE
       wxwidgets::base
)
```

The transitive dependency on lib-utility-interface is now implied, so the generated graph of dependencies can have one less arc.  Consistently with other libraries, only the dependency on wxBase should be private.  Libraries that depend on lib-graphics could themselves simplify their CMakeLists.txt by using transitive dependency on lib-math and lib-screen-geometry.